### PR TITLE
React calculator Math-Magicians

### PR DESCRIPTION
**Pull request by dasileker**
### **React calculator Math-Magicians**

_**In this Milestone:**_

- Add NavBar:
- Add Link:
- Add Routes:
- Add Home page:
- Add Quote page:

**_I used:_**

1. NPM.
2. GIT.
3. JEST.

### _**Live Demo**_

<a href="https://alhasiba.netlify.app/">
  Deploy
</a>

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @dasileker, congratulations on this project :1st_place_medal: The PR is descriptive :+1: The README file is professional :slightly_smiling_face: The commit messages are easier to understand :100: Some of the test cases passed. However, kindly check the comments below and addressed them accordingly:

- Let us add a snapshot test for the following: this [link](https://jestjs.io/docs/snapshot-testing) could help
  - Home
  - Calculator
  - Quote
- Some of our tests are failing, kindly check below to see the files with the failing tests
  - Calculate.test.js, with a TypeError: result.toNumber is not a function and TypeError: big2.toNumber is not a function
  - Operate.test.js, TypeError: result.toNumber is not a function and TypeError: big2.toNumber is not a function
  - Index.test.js, expect(jest.fn()).toHaveBeenCalledWith(...expected)
- Kindly use the right linter configuration for our .github/workflows/linters.yml. You can find it [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml)

[Optional]
- It would be professional if we have all our test in a Test directory

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: src/Components/App.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import calculate from '../logic/calculate';
 import Display from './Display';
+// import NavBar from './Nav';
```
Comment (addod19) [inline src/Components/App.js]: Kindly remove commented code

File: src/Components/Home.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+// import NavBar from './Nav';
+import './App.css';
+
+const Home = () => (
+  <>
+    {/* <div className="home-title">
+      <NavBar />
+    </div> */}
```
Comment (addod19) [inline src/Components/Home.js]: Kindly remove commented piece of code

File: src/Components/Nav.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+// import Home from './Home';
```
Comment (addod19) [inline src/Components/Nav.js]: Kindly remove commented piece of code

**Review by addod19 — CHANGES_REQUESTED** [review]

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**